### DISCUSSION TITLE
help!

### DISCUSSION THREAD

**Discussion by morais1987** [kookehs/bdo-marketplace#18] [General]
Good afternoon!!
I am a Server SA player!
Im trying to make this table with CSV dump, but I can't!
Already got the VerificationToken for cooki and token, Ok!
Only dont know how to do "Execute binary for your system", is there any video teaching or could you record video teaching me please? Thank you very much if possible!

**Comment by kookehs** [comment]
"Execute binary for your system" entails running the program correct for your operating system.
The latest binaries are found [here](https://github.com/kookehs/bdo-marketplace/releases) under `Assets`.
- If you're are on Mac you'll choose one of the `darwin` executables, otherwise `linux` for Linux and `windows` for Windows. Each one has 32-bit and 64-bit support with the exception of Mac which no longer supporting 32-bit.

You'll also need to download `config.json` and paste your tokens in there.
One other file you may want is `items.json` which is a reference for item IDs. You can also create your own and populate it with item IDs from the reference file or a database website.
- It's not recommended to use the default file as it will grab the data for **many** items.
